modules/exploits/windows/http/tomcat_ghostcat.rb

```diff
@@ -0,0 +1,267 @@
+class MetasploitModule < Msf::Auxiliary
```

Comment:
This is causing the tests to fail, it looks like everything else in this folder extends from `Msf::Exploit::Remote` I think that might fit the use case better

Reply:
Dear,
We checked our code and found it to run very well.
The problem if we change `Msf :: Auxiliary` to `Msf :: Exploit :: Remote,`

LHOST, LPORT values will not be the same as the original target of the code.

`Msf :: Auxiliary`
```
Module options (auxiliary/windows/tomcat_ghostcat):

   Name      Current Setting   Required  Description
   ----      ---------------   --------  -----------
   FILENAME  /WEB-INF/web.xml  yes       File name
   PORTWEB                     no        Set a port webserver
   RHOSTS                      yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
   RPORT                       yes       The target port (TCP)
   SSL       false             yes       SSL

```

After changing `Msf :: Exploit :: Remote,`
```
Module options (exploit/windows/tomcat_ghostcat):

   Name      Current Setting   Required  Description
   ----      ---------------   --------  -----------
   FILENAME  /WEB-INF/web.xml  yes       File name
   PORTWEB                     no        Set a port webserver
   RHOSTS                      yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
   RPORT                       yes       The target port (TCP)
   SSL       false             yes       SSL


Payload options (generic/shell_reverse_tcp):

   Name   Current Setting  Required  Description
   ----   ---------------  --------  -----------
   LHOST  192.168.1.4      yes       The listen address (an interface may be specified)
   LPORT  4444             yes       The listen port


Exploit target:

   Id  Name
   --  ----
   0   Automatic


```
This module is not designed for users to enter payload, LHOST, or LPORT

Do you have any suggestions for us? Thank you very much

Reply:
Yes, Thank you so much !!!

Reply:
We have updated our code. Sorry for the problem